velox/type/Variant.h

```diff
@@ -469,6 +545,15 @@ class variant {
     return value<CppToType<T>::typeKind>();
   }
 
+  bool isNull() const {
```

Comment:
Any particular reason for moving this method? Looks like there are other methods that rely on ptr_ null-ness: isSet, checkNotNull, etc. Do these need updating as well?

Reply:
They do indeed. Will fix.

Reply:
I had to move this below the `value()` implementation as it uses the `value()` method.

Reply:
I removed `isSet` since there is a `hasValue()` with the same implementation.